### ISSUE TITLE
expo-updates does not show splashscreen 

### ISSUE THREAD

**Issue by ziyoshams** [expo/expo#16748]
### Summary

After just installing `expo-updates` `react-native-bootsplash`'s splashscreen gets messed up. There is a white splash first  then splashscreen and then the app. Best seen in Release build. 
### without expo modules installed
https://user-images.githubusercontent.com/14338593/160018684-28872ac3-cc2f-4021-9eb6-c62483ef21eb.mp4

### with expo modules installed
https://user-images.githubusercontent.com/14338593/160019147-12661f92-029f-425d-ad3c-bb1b5db5bc1d.mp4

### with expo updates installed
![Simulator Screen Recording - iPhone 11 Pro Max - 2022-03-25 at 03 20 21](https://user-images.githubusercontent.com/14338593/160019744-70e9bae2-ec4a-4775-a5d7-aedc93b0cfbb.gif)

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

iOS

### Environment

```
  expo-env-info 1.0.2 environment info:
    System:
      OS: macOS 12.3
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.13.0 - ~/.nvm/versions/node/v16.13.0/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 8.1.0 - ~/.nvm/versions/node/v16.13.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.2 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
      Android SDK:
        API Levels: 26, 27, 28, 29, 30
        Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2
        System Images: android-22 | Google APIs Intel x86 Atom, android-26 | Intel x86 Atom_64, android-26 | Google APIs Intel x86 Atom, android-27 | Intel x86 Atom_64, android-28 | Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom, android-29 | Intel x86 Atom_64, android-30 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8092744
      Xcode: 13.3/13E113 - /usr/bin/xcodebuild
    npmPackages:
      expo: >=44.0.0-0 <45.0.0 => 44.0.6
      react: 17.0.2 => 17.0.2
      react-native: 0.67.4 => 0.67.4
    npmGlobalPackages:
      eas-cli: 0.48.2
      expo-cli: 5.3.0
    Expo Workflow: bare
```

### Reproducible demo

https://github.com/ziyoshams/expo-updates-splashscreen-issue

**Comment by realtril** [comment]
have you find a way to fix the issue?

**Comment by ziyoshams** [comment]
@realtril Nope. Waiting for expo  to resolve this. Hopefully someone can prioritize this issue? cc @EvanBacon @brentvatne @Kudo 
This is a paid service in the end

**Comment by brentvatne** [comment]
you can work around this issue by using expo-splash-screen instead, or you can use the customized manual setup for expo-updates: https://docs.expo.dev/bare/installing-updates/#customizing-automatic-setup-for-ios

replace `[controller startAndShowLaunchScreen:self.window];` with `[controller start];` and expo-updates won't interfere with the splash screen

**Comment by nouri-soirees** [comment]
expo-updates with react-native-bootsplash still makes the same issue!

**Comment by shoamano83** [comment]
I had a bare project running React Native 0.72.7 and `react-native-bootsplash` 4.7.5. I recently introduced Expo 49 with `expo-updates` 0.18.19 and noticed my splash screen not working on iOS; when I launched my app, it showed the splash screen for a short time, then a white screen was shown until app's main screen came out.

I know latest `react-native-bootsplash` has Expo plugin support which requires Expo 51. Unfortunately my project needs to stick with current RN version so I cannot upgrade Expo.

So I looked into `expo-updates` code to find that it had a flag called `EXUpdatesAutoSetup`. Configuring it to `false` prevented the component from creating rootView and it seemed to resolve my issue.

FYI here's the change I made:

```
diff --git a/ios/myProjectName/Supporting/Expo.plist b/ios/myProjectName/Supporting/Expo.plist
index ffd585b6..4e5dd701 100644
--- a/ios/myProjectName/Supporting/Expo.plist
+++ b/ios/myProjectName/Supporting/Expo.plist
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+       <key>EXUpdatesAutoSetup</key>
+       <false/>
        <key>EXUpdatesCheckOnLaunch</key>
        <string>ALWAYS</string>
        <key>EXUpdatesLaunchWaitMs</key>
```
Note: the flag `EXUpdatesAutoSetup` is outdated and it no longer exists in latest `expo-updates`. So this workaround may not work depending on the version you're using.
